data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -106,16 +106,20 @@
 ENERGY_BURDEN_FIELD = "Energy burden"
 
 # Health
-DIABETES_FIELD = "Diagnosed diabetes among adults aged >=18 years"
-ASTHMA_FIELD = "Current asthma among adults aged >=18 years"
-HEART_DISEASE_FIELD = "Coronary heart disease among adults aged >=18 years"
-CANCER_FIELD = "Cancer (excluding skin cancer) among adults aged >=18 years"
+DIABETES_FIELD = (
+    "Diagnosed diabetes among adults aged greater than or equal to 18 years"
+)
+ASTHMA_FIELD = (
```

Comment:
Posting here as well as in our other conversation to discuss the bug on the field name: 

> this bug was probably introduced by this PR: https://github.com/usds/justice40-tool/pull/1077/files#diff-cc9c42ab3dd9470697a6a2f7b749bb4eef53337ab0c72b10b8669af1e8860396R112.
we rewrote the word “>=” into “greater than or equal to” for a bunch of fields, but some of these come directly from the source data without renaming.
i’d cast a strong preference for option #1 (rename it in the ETL script) out of the two you listed – i’m fairly nervous about introducing an additional renaming in a new place, i think that makes it hard for a developer to understand where different fields are coming from